service/src/test/java/bio/terra/workspace/common/utils/AwsUtilsTest.java

```diff
@@ -52,7 +52,7 @@ void hello_bucket() throws IOException {
     // This lifetime should track that of the discovered environment where possible (stale
     // credentials will get refreshed under the hood).
     AwsCredentialsProvider awsCredentialsProvider =
-        AwsUtils.createWsmCredentialProvider(awsConfiguration, environment);
```

Comment:
please add CRUD tests
- controller test, examples https://github.com/DataBiosphere/terra-workspace-manager/tree/main/service/src/test/java/bio/terra/workspace/app/controller

Reply:
handled in https://verily.atlassian.net/browse/TERRA-306